src/core/highlight-vars.js

```diff
@@ -0,0 +1,102 @@
+
+/**
+ * Module core/highlight-vars
+ * Highlights occurrences of a <var> within a section on click.
+ * Set `conf.highlightVars = true` to enable.
+ * Removes highlights from <var> if clicked anywhere else.
+ * All is done while keeping in mind that exported html stays clean 
+ * on export.
+ */
+import { sub } from "core/pubsubhub";
+import hlVars from "deps/text!core/css/var.css";
+
+export const name = "core/highlight-vars";
+
+export function run(conf) {
+  if (!conf.highlightVars) {
+    return;
+  }
+  const styleElement = document.createElement("style");
+  styleElement.textContent = hlVars;
+  styleElement.classList.add("removeOnSave");
+  document.head.appendChild(styleElement);
+
+  document.addEventListener("click", initHighlight);
```

Comment:
I'm wondering if we shouldn't just attach listener on `<var>`s instead?

Reply:
Just thinking out loud here: 

```JS
[...document.querySelectorAll("var")].forEach(varElem =>
  varElem.addEventListener("click", e => {
    e.stopPropagation();
    highlightVars(e.target);
    const section = e.target.closest("section");
    section.addEventListener("click", resetHighlight, { once: true });
  })
);

function resetHighlight({ target: section }) {
  [...section.querySelectorAll("var.respec-active")].forEach(removeHighlight);
  [...HL_COLORS.keys()].forEach(key => HL_COLORS.set(key, true));
}
```

Reply:
fixed code above... tested it, it works kinda nice :) @sidvishnoi, @saschanaz, thoughts?

Reply:
Actually, need the reset listener on the body instead. Fixed above.